ada/language/ast.py

```diff
@@ -36,6 +37,44 @@ def default_origin():
     return (origin, No(T.AdaNode))
 
 
+def default_imprecise_fallback():
+    """
+    Helper to return an imprecise fallback dynamic param spec which defaults to
+    False.
+    """
+    return (imprecise_fallback, False)
+
+
+def _referenced_decl_internal_impl(subject, try_immediate):
```

Comment:
Plz don't do a macro, we try to avoid them as much as possible and here I don't see any reason that this cannot be a function ?

Reply:
Okay I'll put that in another property.